pages/common/venv.md

```diff
@@ -0,0 +1,16 @@
+# venv
+
+> Create lightweight "virtual environments" in python.
+> More information: <https://docs.python.org/3/library/venv.html>.
+
+- Create a python virtual environment:
+
+` python -m venv {{path/to/new/virtual/environment}}`
```

Comment:
```suggestion
` python -m venv {{path/to/new_venv}}`
```
I think this would be better.

Reply:
Hi
I think `new_venv` can get confused with `venv`.
Can we keep it as `new_virtual_env` ?

Reply:
Sure, I will request a review from other maintainers for a second opinion.

Reply:
> Hi I think `new_venv` can get confused with `venv`. Can we keep it as `new_virtual_env` ?

```suggestion
` python -m venv {{path/to/virtual_environment}}`
```